src/org/jitsi/impl/neomedia/rtp/translator/OutputDataStreamImpl.java

```diff
@@ -205,6 +207,8 @@ private int doWrite(
             OutputDataStreamDesc s = streams.get(i);
             StreamRTPManagerDesc streamRTPManager
                 = s.connectorDesc.streamRTPManagerDesc;
+            // Reset the sequence number to the original value
```

Comment:
Can you please describe in more detail what's going on here (that there's a contract that writing may result in a change the sequence number but not anything else, and what the reasons are). I really think that's a tricky thing and worth documenting.